<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player_informationn</class>
 <widget class="QDialog" name="player_informationn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(55, 0, 60);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/img/athlete.png);
background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player_name_info">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="player_name_info_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Total points :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Total_points_info">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="ALL_GW">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="player_name_info_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>GW points</string>
   </property>
  </widget>
  <widget class="QLabel" name="Gw_points_per_each">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>GW points</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>player_name_info</zorder>
  <zorder>player_name_info_2</zorder>
  <zorder>Total_points_info</zorder>
  <zorder>ALL_GW</zorder>
  <zorder>player_name_info_4</zorder>
  <zorder>Gw_points_per_each</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
